syftbox/client/server_client.py

```diff
@@ -173,6 +174,7 @@ def download_files_streaming(self, relative_paths: list[Path], output_dir: Path)
             "POST",
             "/sync/download_bulk",
             json={"paths": relative_str_paths},
+            headers={HEADER_SYFTBOX_VERSION: __version__},
```

Comment:
isnt this already set in BaseClient?

Reply:
that's what I thought but somehow we don't pass a client from the BaseClient, still looking into this

Reply:
fixed it, the tests were the issue